swap_meet/vendor.py

```diff
@@ -1,2 +1,68 @@
+def check_if_empty(lst):
+    if len(lst) == 0:
+        return False
+    return True
```

Comment:
I like the idea of this! However, the name is a little bit confusing. A common naming practice you may find handy is using "is" for functions that do a boolean check. For example, you could name this one `is_not_empty`.